auth/libraries/pip-auth/authlib/config.py

```diff
@@ -0,0 +1,40 @@
+from authlib.dev.keys import key_dict
+
+public_cookie_key = None
+private_cookie_key = None
+public_token_key = None
+private_token_key = None
+
+
+def is_dev_mode():
+    return True
+
+
+def get_public_token_key():
+    return init_key(public_token_key, "rsa-token.key.pub")
+
+
+def get_public_cookie_key():
+    return init_key(public_cookie_key, "rsa-cookie.key.pub")
+
+
+def get_private_token_key():
+    return init_key(private_token_key, "rsa-token.key")
+
+
+def get_private_cookie_key():
+    return init_key(private_cookie_key, "rsa-cookie.key")
+
+
+def init_key(variable, path):
+    if variable is None:
+        variable = get_file(path)
```

Comment:
this will not cache the variable, since it is a local one, not the global. For this purpose use global dict:

```
keys = {}

def init_key(name, path):
    global keys
    if name not in keys:
        keys[name] = get_file(path)
    return keys[name]
```
where name is just a string, like `"private_cookie_key"`.

Reply:
maybe just analog to the `keys.py` with the same names